core/frontend/src/BackgroundMapGeometry.ts

```diff
@@ -49,12 +49,12 @@ export class BackgroundMapGeometry {
   private static _scratchRayAngles = new Array<LongitudeLatitudeNumber>();
   private static _scratchPoint = Point3d.createZero();
 
-  constructor(private _bimElevationBias: number, globeMode: GlobeMode, iModel: IModelConnection) {
-    this._ecefToDb = iModel.backgroundMapLocation.getMapEcefToDb(_bimElevationBias);
+  constructor(private _bimElevationBias: number, globeMode: GlobeMode, private _iModel: IModelConnection) {

```

Comment:
Why is `_bimElevationBias` now a member? I don't see `this._bimElevationBias` referenced anywhere.

Reply:
@RBBentley - do you know the answer to Paul's queston?

Reply:
It's `private`. Unless I'm overlooking some usage of it there's no reason for it to be a member (and then I'm disappointed the compiler doesn't complain about it).

Reply:
It seems like it used internally; places like the following:

this._ecefToDb = _iModel.backgroundMapLocation.getMapEcefToDb(_bimElevationBias);

this._mercatorFractionToDb = this._mercatorTilingScheme.computeMercatorFractionToDb(this._ecefToDb, _bimElevationBias, _iModel, false);

Reply:
Of course, those are in the constructor, so not necessarily needed to be a member then.

But I do see it used here in a method:

  public getPlane(offset = 0) {
    return Plane3dByOriginAndUnitNormal.create(Point3d.create(0, 0, this._bimElevationBias + offset), Vector3d.create(0, 0, 1))!;
  }

Reply:
I see; it was already a member prior to these changes.
I hate having to consult the constructor to find out what the members of a class are but ok.